enhancements/network/user-defined-network-segmentation.md

```diff
@@ -357,11 +359,106 @@ block/allow cluster egress or ingress traffic will still be enforced.
+These CRDs will be registered by Cluster Network Operator (CNO). 
+
+#### CRDs
+
+Following the [CRDs for Managing Networks](#crds-for-managing-networks), two CRDs shall be introduced: 
+- Namespace scoped CRD - represent user request for creating namespace scoped OVN network.
+  - Shall be defined with `namespace` scope.
+- Cluster scoped CRD - represent user request for creating cluster scoped OVN network, enables cross-namespace networking.
+  - Shall be defined with `cluster` scope
+
+The CRD instance There should be will be a finalizer on the CRDs, so that upon deletion OVN-Kubernetes can validate that there are no pods still using this network.
+If there are pods still attached to this network, the network will not be removed.
+
+##### spec
+
+The CRDs spec defines as follows:
+
+| Field name     | Description                                                                                                                                                                                                                                                                                                                                                                                        | optional |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| Topology       | The topological configuration for the network. Must be one of `layer2` or `layer3`.                                                                                                                                                                                                                                                                                                                | No       |
+| Role           | Select the network role in the pod, either `primary` or `secondary`.                                                                                                                                                                                                                                                                                                                               | No       |
+| MTU            | The maximum transmission unit (MTU).<br/>The default value, 1300, is automatically set by the kernel                                                                                                                                                                                                                                                                                               | Yes      |
```

Comment:
The default in OVNK is 1400 bytes, which is a defaulted config parameter and not determined by the kernel.